src/enforcers/NoCalldataEnforcer.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT AND Apache-2.0
+pragma solidity 0.8.23;
+
+import { ExecutionLib } from "@erc7579/lib/ExecutionLib.sol";
+import { ModeLib } from "@erc7579/lib/ModeLib.sol";
+
+import { CaveatEnforcer } from "./CaveatEnforcer.sol";
+import { Execution, ModeCode } from "../utils/Types.sol";
+import { CALLTYPE_SINGLE, CALLTYPE_BATCH } from "../utils/Constants.sol";
+
+/**
+ * @title NoCalldataEnforcer
```

Comment:
I see some places where `calldata` is all lowercase but in others it is `callData`